ui.content/src/main/content/jcr_root/etc/map.publish/https/wknd-site/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="sling:Folder"
+    jcr:mixinTypes="[sling:MappingSpec]"
+    sling:internalRedirect="[/,/content/wknd]"
+    sling:match="$[env:AEM_DOMAIN_WKND_SITE].\\d+"/>
```

Comment:
@Buuhuu if someone wanted to mimic this for an on-prem (6.5) deployment they would need to use some other mechanism, yes?

Reply:
FWIW I think WKND should be cloud first and backwards compatible whenever possible, but just wanted to know

Reply:
Using environment variables works as well for AEM on-prem and AMS. However, afaik it is currently only possible with CM and AEMaaCS to set the environment variables in a self-service way. 

More details can be found here in the [Apache Sling Mappings for Resource Resolution documentation](https://sling.apache.org/documentation/the-sling-engine/mappings-for-resource-resolution.html#string-interpolation-for--etc-map). It would also be possible to refer to `config` in combination with an OSGI configuration in on-prem setups instead of using `env`.